tests/installation/select_patterns.pm

```diff
@@ -107,6 +102,11 @@ sub run {
             };
             assert_screen 'current-pattern-selected', 5;
         }
+
+        # stick to the default patterns
+        if (get_var('PATTERNS', '') =~ /default/) {
+            $needs_to_be_selected = $selected;
```

Comment:
I never understood why there is elsif, why it's not just if ?
allpatterns, minimal,base or default,smt does work with this broken logic :P

allpatterns
http://10.100.98.90/tests/1733/modules/installation_overview/steps/1
base,minimal
http://10.100.98.90/tests/1734/modules/installation_overview/steps/1
default,smt
http://10.100.98.90/tests/1732/modules/installation_overview/steps/1